docs/zh_CN/2023-02-14-so-whats-next.md

```diff
@@ -108,9 +108,9 @@ core-js不仅仅是在网站的前端——它几乎在所有使用JavaScript的
 
 2012年，我把我的开发栈切换到了全栈JavaScript。当时JavaScript仍然太原始——IE仍然比其他任何东西都更受欢迎，ES3时代的浏览器仍然占据了Web的主要部分，最新的NodeJS版本是0.7——它才刚刚开始。JavaScript 仍然不适用于编写严肃的应用程序，开发人员用 CoffeeScript 语言转译器，解决了JavaScript所缺乏的语法糖问题，用 Underscore 等解决了 Javascript 缺乏标准库的问题。 然而，他们并不是标准，随着时间的推移，这些语言和库连同使用它们的项目一起过时了。因此，我满怀希望地期待即将到来的ECMAScript Harmony 6标准。
 
-旧JavaScript引擎仍然流行，用户并不着急，还没有什么机会放弃它们，即使新的ECMAScript标准有着快速和可靠的优点，想让JavaScript引擎支持这个新标准，也要等很多年。但可以使用一些工具尝试使用新标准，让转译器Transpiler（这个词不像现在这么流行）和标准库来解决语法和polyfill的问题。当时，这些工具包才刚刚出现。
+旧JavaScript引擎仍然流行，用户并不着急，还没有什么机会放弃它们，即使新的ECMAScript标准有着快速和可靠的优点，想让JavaScript引擎支持这个新标准，也要等很多年。但可以使用一些工具尝试使用新标准，让转译器（这个词不像现在这么流行）和标准库来解决语法和polyfill的问题。当时，这些工具包才刚刚出现。
```

Comment:
When Chinese and English words are in the same sentence, there should be a space between the Chinese and English words.

```suggestion
旧 JavaScript 引擎仍然流行，用户并不着急，还没有什么机会放弃它们，即使新的 ECMAScript 标准有着快速和可靠的优点，想让 JavaScript 引擎支持这个新标准，也要等很多年。但可以使用一些工具尝试使用新标准，让转译器（这个词不像现在这么流行）和标准库来解决语法和 polyfill 的问题。当时，这些工具包才刚刚出现。
```